pkg/hintrunner/zero/zerohint_blake_test.go

```diff
@@ -427,5 +427,98 @@ func TestZeroHintBlake(t *testing.T) {
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(4294967295),
+						feltUint64(813310313), feltUint64(2491453561), feltUint64(3491828193), feltUint64(2085238082),
+						feltUint64(1219908895), feltUint64(514171180), feltUint64(4245497115), feltUint64(4193177630),
+						feltUint64(1795745351), feltUint64(3144134277), feltUint64(1013904242), feltUint64(2773480762),
+						feltUint64(1359893119), feltUint64(2600822924), feltUint64(528734635), feltUint64(1541459225),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(4294967295), feltUint64(813310313), feltUint64(2491453561),
+						feltUint64(3491828193), feltUint64(2085238082), feltUint64(1219908895), feltUint64(514171180),
+						feltUint64(4245497115), feltUint64(4193177630), feltUint64(1795745351), feltUint64(3144134277),
+						feltUint64(1013904242), feltUint64(2773480762), feltUint64(1359893119), feltUint64(2600822924),
+						feltUint64(528734635), feltUint64(1541459225), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(0),
+						feltUint64(0), feltUint64(0), feltUint64(0), feltUint64(4294967295),
+						feltUint64(813310313), feltUint64(2491453561), feltUint64(3491828193), feltUint64(2085238082),
+						feltUint64(1219908895), feltUint64(514171180), feltUint64(4245497115), feltUint64(4193177630),
+					}),
+			},
+		},
+		"Blake2sFinalize check assert N_PACKED_INSTANCES": {
+			{
+				operanders: []*hintOperander{
+					{Name: "blake2s_ptr_end", Kind: apRelative, Value: addrWithSegment(1, 7)},
+					{Name: "N_PACKED_INSTANCES", Kind: fpRelative, Value: feltUint64(20)},
+					{Name: "INPUT_BLOCK_FELTS", Kind: apRelative, Value: feltUint64(16)},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newBlake2sFinalizeHint(ctx.operanders["blake2s_ptr_end"], ctx.operanders["N_PACKED_INSTANCES"], ctx.operanders["INPUT_BLOCK_FELTS"])
+				},
+				errCheck: errorTextContains("in range [0, 20), got 20"),
+			},
+		},
+		"Blake2sFinalize check assert INPUT_BLOCK_FELTS": {
+			{
+				operanders: []*hintOperander{
+					{Name: "blake2s_ptr_end", Kind: apRelative, Value: addrWithSegment(1, 7)},
+					{Name: "N_PACKED_INSTANCES", Kind: fpRelative, Value: feltUint64(7)},
+					{Name: "INPUT_BLOCK_FELTS", Kind: apRelative, Value: feltUint64(200)},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newBlake2sFinalizeHint(ctx.operanders["blake2s_ptr_end"], ctx.operanders["N_PACKED_INSTANCES"], ctx.operanders["INPUT_BLOCK_FELTS"])
+				},
+				errCheck: errorTextContains("in range [0, 100), got 200"),
```

Comment:
could you add the entire errorText in the errCheck using formatted string?

Reply:
you can leave the 0 and 100 and place 200 as variable in formatted string, no need to pass "[0, 100)" as separate argument to formatted string